build-image/Dockerfile

```diff
@@ -4,6 +4,11 @@ RUN apt-get update && apt-get install -y python-requests python-yaml file jq unz
 RUN go clean -i net && \
 	go install -tags netgo std && \
 	go install -race -tags netgo std
+RUN mkdir -p /go/src/github.com/mvdan && \
+	cd /go/src/github.com/mvdan && \
+	git clone --branch v1.3.0 --single-branch https://github.com/mvdan/sh.git >/dev/null 2>&1 && \
+	cd sh/cmd/shfmt && \
+	go install
```

Comment:
Snap! You beat [me](https://github.com/weaveworks/cortex/pull/434#issuecomment-302712759) to it.